lib/graphql.rb

```diff
@@ -66,6 +66,9 @@ def self.scan(graphql_string)
   def self.scan_with_ragel(graphql_string)
     GraphQL::Language::Lexer.tokenize(graphql_string)
   end
+
+  NOT_CONFIGURED = Object.new
```

Comment:
So this is kind of ROFLscale territory, but I wonder if all these extra references might slow down major GC pause.

One way to avoid this would be to use an immediate object, like a static symbol, e.g. `:__GraphQL__NOT_DEFINED__`. This allows the GC to deretctly skip that reference.

It's not as hack proof, as it's much easier for users to pass that symbol, but I can't imagine it happening by accident.

That said, I'm no GC expert, it might not make a difference at all.

Reply:
I'm going to leave it as-is 😅